libs/langchain/langchain/document_loaders/docugami.py

```diff
@@ -30,22 +29,51 @@
 class DocugamiLoader(BaseLoader, BaseModel):
     """Load from `Docugami`.
 
-    To use, you should have the ``lxml`` python package installed.
+    To use, you should have the ``dgml-utils`` python package installed.
     """
 
     api: str = DEFAULT_API_ENDPOINT
     """The Docugami API endpoint to use."""
 
     access_token: Optional[str] = os.environ.get("DOCUGAMI_API_KEY")
     """The Docugami API access token to use."""
+
+    max_text_length = 1024 * 4
```

Comment:
could we just be 4096

Reply:
Sure! sorry probably a style quirk I always try to do it this way to remember that this is 4k etc :)